### PR TITLE
[image_view] Add colormap option in video_recorder

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

File: image_view/src/nodes/video_recorder.cpp [hunk 1]
```diff
@@ -106,9 +108,11 @@ int main(int argc, char** argv)
     local_nh.param("fps", fps, 15);
     local_nh.param("codec", codec, std::string("MJPG"));
     local_nh.param("encoding", encoding, std::string("bgr8"));
+    // cv_bridge::CvtColorForDisplayOptions
```
Comment (vrabaud) [inline image_view/src/nodes/video_recorder.cpp:113]: useless

File: image_view/src/nodes/video_recorder.cpp [hunk 2]
```diff
@@ -106,9 +108,11 @@ int main(int argc, char** argv)
...
     local_nh.param("min_depth_range", min_depth_range, 0.0);
-    local_nh.param("max_depth_range", max_depth_range, 5.5);
+    local_nh.param("max_depth_range", max_depth_range, 0.0);
```
Comment (vrabaud) [inline image_view/src/nodes/video_recorder.cpp:115]: so, why not 5.5 ?

**Comment by vrabaud** [comment]
please rebase

**Comment by wkentaro** [comment]
Rebased.

**Pull request merged**